apps/req.c

```diff
@@ -545,6 +545,15 @@ int req_main(int argc, char **argv)
                     goto end;
                 }
             }
+
+#ifndef OPENSSL_NO_EC
+            if (pkey_type == EVP_PKEY_EC
+                    && EVP_PKEY_CTX_get_ec_keygen_bits(genctx, &newkey) <= 0) {
```

Comment:
It seems excessive to me to add a whole new API for this, especially this late in the day for 1.1.0. Why don't we just not print the number of bits when its an EC key?